packages/atclient/src/atclient.c

```diff
@@ -369,6 +383,7 @@ exit: {
   free(root_cmd);
   free(from_cmd);
   free(pkam_cmd);
+  free(atserver_host);
```

Comment:
Note to self: potential bug, for me to verify later.

Reply:
fixed in [d78b8fd](https://github.com/atsign-foundation/at_c/pull/400/commits/d78b8fd277257c096d2479fbdd8785dc430301d3)